source/connect/client.txt

```diff
@@ -0,0 +1,110 @@
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you must create the following items:
+
+- **Connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- **MongoDB\\Client** object, which creates the connection to the MongoDB deployment
+  and lets you perform operations on it.
+
+You can also set options within either or both of these components to
+customize the way that the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide describes the components of a connection string and shows how to
+ use a ``MongoDB\Client`` object to connect to a MongoDB deployment.
+
+.. _php-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``db_username:db_password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+       For more information about the ``authSource`` connection option, see
+       :ref:`php-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``db_username:db_password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`php-connection-options` for a full description of
+       these options.
+
+To learn more about connection strings, see
+:manual:`Connection Strings </reference/connection-string>` in the
+Server manual.
+
+Create a MongoDB\Client
+-----------------------
+
+To create a connection to MongoDB, pass your connection string when constructing
+an instance of the ``MongoDB\Client`` class.
+
+In the following example, the library uses a sample connection URI to connect to a MongoDB 
+deployment on port ``27017`` of ``localhost``:
+
+.. literalinclude:: /includes/connect/client.php
+   :language: php
+   :copyable: true
+
+.. tip:: Reuse Your Client
+
+   Because each ``MongoDB\Client`` object represents a pool of connections to the
+   database, you can use a single instance of the class across multiple requests.
```

Comment:
To be checked by @jmikola @alcaeus, but I don't think there's a pool of connections, since requests are blocking and PHP is monothread.

Reply:
That is correct, PHP does not implement a connection pool. We have an [article in the extension docs](https://www.php.net/manual/en/mongodb.connection-handling.php) that details the connection persistence mechanism used by the PHP driver (current and the legacy one used before 1.2.0).